tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-distributed-test-ci-pipeline.yml

```diff
@@ -53,3 +53,51 @@ jobs:
       condition: 'succeeded'
 
   - template: templates/clean-agent-build-directory-step.yml
+
+- job: Onnxruntime_Linux_GPU_Inference_Distributed_Test
```

Comment:
You may put the two jobs in two stages, like what is in https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/azure-pipelines/post-merge-jobs.yml . Then when someone manually run this pipeline, they can choose to only run one of the jobs from Azure Devops' web UI.

Reply:
thanks!